react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from 'react';
+
+import { randomInt } from '../../base/util';
+
+import { reconnectNow } from '../functions';
+import ReloadButton from './ReloadButton';
+
+declare var AJS: Object;
+declare var APP: Object;
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Implements abstract React Component for the page reload overlays.
+ */
+export default class AbstractPageReloadOverlay extends Component {
+    /**
+     * AbstractPageReloadOverlay component's property types.
+     *
+     * @static
```

Comment:
Actually I checked now and most of the code in react directory has empty line between the description and the modifiers. Also for methods eslint complains. I added empty lines everywhere.